shell/browser/hid/electron_hid_delegate.cc

```diff
@@ -82,21 +85,28 @@ bool ElectronHidDelegate::HasDevicePermission(
     content::BrowserContext* browser_context,
     const url::Origin& origin,
     const device::mojom::HidDeviceInfo& device) {
-  return GetChooserContext(browser_context)
-      ->HasDevicePermission(origin, device);
+  auto* chooser_context = GetChooserContext(browser_context);
+  if (!chooser_context)
+    return false;
+  return chooser_context->HasDevicePermission(origin, device);
 }
 
 void ElectronHidDelegate::RevokeDevicePermission(
     content::BrowserContext* browser_context,
     const url::Origin& origin,
     const device::mojom::HidDeviceInfo& device) {
-  return GetChooserContext(browser_context)
-      ->RevokeDevicePermission(origin, device);
+  auto* chooser_context = GetChooserContext(browser_context);
+  if (!chooser_context)
+    return;
+  chooser_context->RevokeDevicePermission(origin, device);
 }
 
 device::mojom::HidManager* ElectronHidDelegate::GetHidManager(
     content::BrowserContext* browser_context) {
-  return GetChooserContext(browser_context)->GetHidManager();
+  auto* chooser_context = GetChooserContext(browser_context);
+  if (!chooser_context)
+    return nullptr;
+  return chooser_context->GetHidManager();
```

Comment:
We should more closely mirror upstream here, eg:
```suggestion
  if (!browser_context) {
    return nullptr;
  }
  return GetChooserContext(browser_context)->GetHidManager();
```